.kubectl-aliases.sh

```
@@ -0,0 +1,74 @@
+kgn() {
+    echo "FRIENDLY   SHORT-ID  STATUS    CPU       MEMORY"
+    echo "------------------------------------------------------------"
+    kubectl get nodes -o json 2>/dev/null | jq -r '.items[] | 
+        "\(.metadata.labels."cluster.pixelated.io/friendly-name" // "unknown")|\(.metadata.labels."cluster.pixelated.io/short-id" // "unknown")|\(([.status.conditions[] | select(.type=="Ready")] | first | .status) // "Unknown")|\(.status.allocatable.cpu // "unknown")|\(.status.allocatable.memory // "unknown")"' 2>/dev/null | \
+    while IFS='|' read -r friendly short node_status cpu mem; do
+        printf "%-10s %-8s %-8s %-9s %s\n" "${friendly}" "${short}" "${node_status}" "${cpu}" "${mem}"
+    done
+}
+
+kgp() {
+    local namespace="${1:-}"
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)